docs/source/branch.rst

```diff
@@ -8,125 +8,167 @@ The Branches File
+    how to parse.
 
 Single Parameter Variation
 ^^^^^^^^^^^^^^^^^^^^^^^^^^
-In order to illustrate the variation of a single parameter, let's assume you have defined a model specification file
-that includes a dietary intervention of egg supplementation and that this intervention is parameterized by the
-proportion of the population that is recruited into the intervention program. We may want to run simulations on several
-different proportions including full recruitment and no recruitment, which would function as a baseline. We can easily
-do this with the following branches file.
+
+In order to illustrate the variation of a single :term:`parameter<Configuration Parameter>`, let's assume
+you have defined a :term:`model specification<Model Specification>` that includes a dietary intervention of
+egg supplementation and that this intervention is parameterized by the proportion of the population that is recruited
+into the intervention program. We may want to run simulations on several different proportions including full
+recruitment and no recruitment, which would function as a baseline. We can easily do this with the following
+branches file.
 
 .. code-block:: yaml
+    :caption: egg_intervention_branches.yaml
 
     branches:
             - egg_intervention:
                     recruitment:
                         proportion: [0.0, 0.4, 0.8, 1.0]
 
-The ``branches`` block specifies changes to values found in the model specification YAML, exactly matching the blocks
-from that specification underneath.  Here, the YAML list [0.0, 0.4, 0.8, 1.0] dictates specific recruitment proportions
-to be simulated. Thus, you can expect four separate simulations to be run, one for each.
+The ``branches`` block specifies changes to values found in the configuration block of the original model specification
+YAML. The block found in the branches file must exactly match the block from the original model specification.
+Here, the YAML list [0.0, 0.4, 0.8, 1.0] dictates specific recruitment proportions to be simulated.
+Thus, you can expect four separate simulations to be run, one for each variation.
 
 .. warning::
+
     Varying the time step, start or end time, or the population size of a simulation will make profiling very difficult
     and runs the risk of breaking our output writing tools.
 
 
 Interaction with Uncertainty
 ^^^^^^^^^^^^^^^^^^^^^^^^^^^^
-As touched upon in the section on :ref:`combining draws and seeds<Combining Draws and Seeds>`, when multiple branch
-configurations would result in multiple simulations, the result is a simulation for every combination, or the cartesian
-product, of the parameters. Let's add draws to our previous branches file and figure out how many simulations it will
-result in.
+As touched upon in the section on :ref:`combining draws and seeds<Combining Draws and Seeds>`, when multiple
+:term:`branch configurations<Branch Configuration>` would result in multiple simulations the result is a simulation for
```

Comment:
I'm not sure you want branch configurations here - the glossary defines the branch config as the single branches yaml file but this sentence seems to be using it to refer to parameters within that file

Reply:
yes maybe configuration parameter. Really it is when different configurations would lead to "branches"